vagrant/vagrant.config.dist

```diff
@@ -0,0 +1,19 @@
+# Copy this file to `vagrant.config` and change values to keep them oer a `git pull`
+
+# The name of your Freifunk community.
+INSTANCE_NAME="Freifunk Alpha Centauri"
+
+# The sender address of the emails that are sent by ff-node-monitor.
+EMAIL_FROM="freifunk@alpha-centauri.mw"
+
+# The URL to the hopglass nodes.json file.
+NODES_URL="https://map.freifunk.in-kiel.de/yanic/nodes.json"
```

Comment:
So you are fine with the test setups all going to your server for their JSON data?